deployer/scripts/install.sh

```diff
@@ -209,7 +208,27 @@ function generate_secrets() {
       done
     fi
 
-  fi # supporting infrastructure - secrets
+    generate_configmaps
+}
+
+function generate_configmaps() {
+    ### ConfigMaps
+    echo "Deleting configmaps"
+    oc delete configmap -l logging-infra=support
+
+    echo "Creating configmaps"
+
+    # generate elasticsearch configmap 
+    oc create configmap logging-elasticsearch \
```

Comment:
I think if we have an upgrade that requires a config change, we have to clobber any changes they may have made. I'm not thinking that we'll encourage customers to change their configmaps; that would be more a debug capability. If there's a special case where they really do need to change them for some legitimate function, they'll have to put their changes back in place after the hypothetical upgrade.